internal/bffauth/client.go

```diff
@@ -8,13 +8,21 @@ type ExternalHabitatUser struct {
 	Host      string
 }
 
-type Client struct {
+type Client interface {
+	GetToken(did string) (string, error)
+}
+
+type client struct {
```

Comment:
nit: I like `clientImpl`

Reply:
I think ethan is changing this so i'm going to leave it